code/game/mecha/mech_fabricator.dm

```diff
@@ -135,7 +150,35 @@
 			update_static_data(user)
 		return
 	return ..()
-
+/**
+ * All the negative wire effect
+ * Zap doesnt actually zap, it breaks one limb (Because pain is to be had)
```

Comment:
dont call it zap then

Reply:
Was like that for consistent naming with the wire as defining a new one would be a bit too needless. Changed anyway